PlaceRouteHierFlow/Dockerfile

```diff
@@ -11,4 +11,4 @@ ENV LD_LIBRARY_PATH=/usr/local/lib/lpsolve/lp_solve_5.5.2.5_dev_ux64
 WORKDIR /PlaceRouteHierFlow
 
 RUN \
-    make
+    make GOOGLETEST_UNSUPPORTED_BUILD_SYSTEM_WORKAROUND=1
```

Comment:
This appears to break CI. Everything else looks good !

Reply:
That is not what broke CI. That was my attempt to fix CI. A change in googletest caused the break. I'm fixing that. Can you approve the review so I can merge the PR when I get that fixed.

Reply:
Done !